### DISCUSSION TITLE
Whisper add Urdu (Pakistan) language

### DISCUSSION THREAD

**Discussion by Aamirfarooq13** [openai/whisper#2268] [General]
I'm a iPhone user, ChatGPT app not have option to select Urdu language for speech detection, my request if OpenAI using whisper technology into ChatGPT 4o then please add Urdu language, thank you

**Comment by glangford** [comment]
fyi the whisper table of available languages lists Urdu already

https://github.com/openai/whisper?tab=readme-ov-file#available-models-and-languages

also in the software itself 'ur' is a supported language code. 

For ChatGPT, you need to raise any questions you have over in the OpenAI developer forum, not in this open source whisper discussion area. See for example

https://community.openai.com/t/misunderstanding-the-language/712567
